src/PackageManagement/BuildIntegration/BuildIntegratedRestoreUtility.cs

```diff
@@ -90,5 +91,29 @@ public static class BuildIntegratedRestoreUtility
 
             return result;
         }
+
+        /// <summary>
+        /// Find all packages added to <paramref name="updatedLockFile"/>.
+        /// </summary>
+        public static IReadOnlyList<PackageIdentity> GetAddedPackages(LockFile originalLockFile, LockFile updatedLockFile)
+        {
+            var updatedPackages = updatedLockFile.Targets.SelectMany(target => target.Libraries)
```

Comment:
It shouldn't matter, this purposely ignores targets since if it is used in any form we need to run the init.ps1